public/Lab-Files/Lesson03/Challenges/box-positioning.css

```diff
@@ -0,0 +1,54 @@
+.box {
+	width: 50px;
+	height: 50px;
+	margin:30px;
+	padding:10px;
+	border:1px;
+	font-style: bold;
+}
+
+.box1 { 
+		background: black; 
+		float: left;
+		clear: left;
+		color:white;
+}
+
+.box2{ 
+		background: white; 
+		float: left;
+		clear: right;
+		border-color: black;
+		border-style: solid;
+}
+
+.box3{ 
+		background: black; 
+		float: left;
+		clear: right;
+		color:white;
+}
+
+.box4{ 
+		background: white; 
+		float: left;
+		clear: right;
+		border-style: solid;
+		clear: both;
+}
+
+.box5{ 
+		background: black; 
+		float: left;
+		clear: right;
+		color:white;
+}
+
+.box6{ 
+		background: white; 
+		float: left;
+		clear: right;
+		border-style: solid;
+}
+
```

Comment:
Manny, 

Roberto posted in rocket chat a good solution to this challenge, and the CSS is much shorter. Take a look at his solution :)